src/core/storage/saved-edits.ts

```diff
@@ -12,8 +12,8 @@ class SavedEditsImpl extends SavedEditsModel {
 		BrowserStorage.LOCAL_CACHE
 	);
 
-	constructor() {
-		super(BrowserStorage.getLocalStorage(BrowserStorage.LOCAL_CACHE));
+	public init() {
+		this._init(BrowserStorage.getLocalStorage(BrowserStorage.LOCAL_CACHE));
```

Comment:
Should we protect if it's already been init?

Reply:
There shouldn't be any harm to running init twice except that you initialize the storage twice.
Though it is only ever run at the very beginning.
_init is protected.